conformance_tests/sysman/test_sysman_power/src/test_sysman_power.cpp

```diff
@@ -894,44 +901,50 @@ TEST_F(
              << _ze_result_t(ZE_RESULT_ERROR_UNSUPPORTED_FEATURE);
     }
     uint32_t count_power = 0;
-    zes_power_limit_ext_desc_t power_peak_initial = {};
-    zes_power_limit_ext_desc_t power_peak_Max = {};
-    zes_power_limit_ext_desc_t power_peak_getMax = {};
-    zes_power_limit_ext_desc_t power_peak_get = {};
+    uint32_t single_count = 1;
+    
     auto power_limits_descriptors =
         lzt::get_power_limits_ext(power_card_handle, &count_power);
     for (int i = 0; i < power_limits_descriptors.size(); i++) {
+      zes_power_limit_ext_desc_t power_peak_initial = {};
+      zes_power_limit_ext_desc_t power_peak_Max = {};
+      zes_power_limit_ext_desc_t power_peak_getMax = {};
+      zes_power_limit_ext_desc_t power_peak_get = {};
       if (power_limits_descriptors[i].level == ZES_POWER_LEVEL_PEAK) {
```

Comment:
Similarly info log can be added if Peak level is not available after iterating over all descriptors

Reply:
done